docs/source/include/learning.rst

```diff
@@ -0,0 +1,93 @@
+Learned Models
+==============
+
+In SCICO, neural network models are used to represent imaging problems and provide different modes of data-driven regularization.
+The models are implemented in `FLAX <https://flax.readthedocs.io/>`_, and constitute a representative sample of frequently used networks.
+
+
+FlaxMap
+-------
+
+SCICO interfaces with the implemented models via :class:`.FlaxMap`. This provides a standardized access to all trained models via the model definiton and the learned parameters. Further specialized functionality, such as learned denoisers, are built on top of :class:`.FlaxMap`. The models implemented are described next.
+
+
+
+DnCNN
+-----
+
+The denoiser convolutional neural network model (DnCNN), :class:`.DnCNNNet`, from :cite:`zhang-2017-dncnn` is used to denoise images that have been corrupted with additive Gaussian noise.
+
```

Comment:
Suggested: ```The denoiser convolutional neural network model (DnCNN) :cite:`zhang-2017-dncnn`, implemented as :class:`.DnCNNNet`, is ...```

Same format for corresponding ODP sentence below.

Reply:
Descriptions have been formated as suggested.